templates/components/entry.html.twig

```diff
@@ -49,9 +49,9 @@
                         </h2>
                     {% endif %}
                 {% elseif(entry.visibility is same as 'trashed') %}
-                    <p class="text-muted">{{ 'deleted_by_moderator' }}</p>
+                    <p class="text-muted font-italic">&lsqb;<i>{{ 'deleted_by_moderator'|trans }}</i>&rsqb;</p>
```

Comment:
`font-italic` might be leftover here? Looks like is isn't a class in any `scss` file so might be able to remove it, as it's not on any of the others